packages/create-ship-app/src/scripts/frontend.sh

```diff
@@ -12,14 +36,29 @@ cd "$project_name"
 git clone --quiet --filter=blob:none --no-checkout --depth 1 --sparse https://github.com/paralect/ship.git
 cd ship
 git sparse-checkout init --cone
-git sparse-checkout add services/web
+git sparse-checkout add services/web deploy
 git checkout
 cd ../
 
 cp -a ship/services/web/. .
 
+if [ ! -z "$platform_common_dir" ]; then
```

Comment:
Why not just copying single file file from the workflows folder?
We always use digital ocean apps deployment type for frontend and backend build types. If it evolves in the future, we will update scripts, at the moment this solution looks too tough. You can avoid `installService` and `cleanup.sh `script.

1. copy single workflow file from `workflows` folder
2. remove deploy folder